### DISCUSSION TITLE
[Feature Request] Allowing different part cooling fan speeds for ironing

### DISCUSSION THREAD

**Discussion by ErrorMsg404** [OrcaSlicer/OrcaSlicer#6088] [Ideas]
I've found that the airflow over a surface is pretty important when ironing.

I'm currently printing a part and the left part of the picture is yet to be ironed, the middle part is ironed with the part fan off and the right part is ironed with the part fan on at 60%.
![20240710_171149](https://github.com/SoftFever/OrcaSlicer/assets/92689821/0cb8468d-171e-4ce3-83b2-3cc5c2538f83)

Hopefully this makes a decent case for why it would be a useful feature to have.

**Comment by TheDIYGuy999** [comment]
I find this useful as well
